data/common.yaml

```diff
@@ -11,6 +11,6 @@ profile_allow_ssh_from_bastion::custom_cfg:
   PasswordAuthentication: "yes"
   PubkeyAuthentication: "no"
 profile_allow_ssh_from_bastion::groups:
-  - "org_asd"
+  - "org_ici"
```

Comment:
There a reason this module is getting `org_asd` replaced with `org_ici` and not `grp_asd_admin, org_cs, org_sp` like the others?

Reply:
Yes. This module is not intended to be specific to 'ASD' usage. I do not think anything related to ASD should be left as a default.

I'm not sure what the default should be, but my gut would be to lean toward allowing all of ICI have access to our resources.

But we could also remove all org groups except org_ici if that makes more sense.

Reply:
Ah gotcha that makes sense.